app/scripts/views/confirm.js

```diff
@@ -60,14 +76,31 @@ define(function (require, exports, module) {
       });
     },
 
-    _gmailTabOpened: function () {
-      this.logViewEvent('openGmail.clicked');
+    _getMissingSessionTokenScreen: function () {
+      var screenUrl = this._isSignUp() ? 'signup' : 'signin';
+      return this.broker.transformLink(screenUrl);
+    },
+
+    _isSignUp: function () {
+      return VerificationReasons.is(this.model.get('type'), 'SIGN_UP');
+    },
+
+    _isSignIn: function () {
+      return VerificationReasons.is(this.model.get('type'), 'SIGN_IN');
+    },
+
+    _navigateToCompleteScreen: function () {
```

Comment:
Noticed that isSignUp, isSignIn, navigateToCompleteScreen functions are reused in `confirm.js` and `complete_sign_in.js`. Is it possible to put these in a shared base view?